### PR TITLE
[gtk] Small Gir.toml cleanup

### PR DESCRIPTION
**Pull request by sophie-h**

### CODE REVIEW

File: gtk/Gir.toml [hunk 1]
```diff
@@ -411,7 +411,6 @@ generate_builder = true
 name = "Gtk.Arrow"
 #deprecated since 3.14
 status = "ignore"
-ignore_builder = true
```
Comment (bilelmoussaoui) [inline gtk/Gir.toml:414]: can't the type just be moved to the generic `ignore = [ ]`?

Comment (sophie-h) [reply gtk/Gir.toml:414]: Good idea. Did a small cleanup.

**Review by sdroege — APPROVED** [review]

**Comment by sdroege** [comment]
@GuillaumeGomez The toml checker ([this](https://github.com/gtk-rs/checker) one I guess) doesn't handler comments correctly :upside_down_face:

**Review by bilelmoussaoui — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Oh indeed. Writing a fix then we can restart the CI here.

Thanks @sophie-h !

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
https://github.com/gtk-rs/gir-format-check/pull/3 and I published a new version so let's retry!

**Comment by GuillaumeGomez** [comment]
So it restarted all tests except the one failing... Weird.

**Comment by sophie-h** [comment]
@GuillaumeGomez Don't we need a rebase for the fix to apply anyway?

**Comment by GuillaumeGomez** [comment]
@sophie-h No, it's referenced by a crate number. And since it's a minor release, we should have it. No need for you to do anything more. :)

**Comment by GuillaumeGomez** [comment]
```
connection error: Connection reset by peer (os error 104)
```

Well, we're getting closer! Restarted the CI once again... It'd be really nice if github allowed to only restart the failing one...

**Pull request merged**